doc/design.txt

```diff
@@ -210,15 +210,15 @@ An image trailer has the following structure:
      0                   1                   2                   3
```

Comment:
I think I agree with you. I'll give it a try on mapping this in a less loose way...